### ISSUE TITLE
delayed_node connects to the P2P network

### ISSUE THREAD

**Issue by vikramrajkumar** [bitshares/bitshares-core#104]
_From @xeroc on October 22, 2015 7:43_

People need to run this

```
./programs/delayed_node/delayed_node --trusted-node="127.0.0.1:8090" -d delayed_node -s "0.0.0.0:0" --p2p-endpoint="0.0.0.0:0" --seed-nodes "[]"
```

in order for the delayed node not to connect to the internet via P2P .. the default seed nodes are in place otherwise

_Copied from original issue: cryptonomex/graphene#388_

**Comment by oxarbitrage** [comment]
https://github.com/bitshares/docs.bitshares.org/pull/26

**Comment by oxarbitrage** [comment]
same is valid for a private testnet setup. for this reason i submitted a new pull request to the documentation in the private testnet setup section:
https://github.com/bitshares/docs.bitshares.org/pull/37

and also updated the private testnet wiki to avoid connecting to default seed nodes hardcoded:

https://github.com/bitshares/bitshares-core/wiki/private-testnet#starting-block-production

closing issue.

**Comment by abitmore** [comment]
What if we disable p2p entirely if `delayed_node` plugin is enabled?

**Comment by abitmore** [comment]
Fixed by #2040 and #2393.
